Gemfile

```diff
@@ -55,3 +55,4 @@ gem 'gravtastic'
 gem 'pickadate-rails'
 gem 'jquery-turbolinks'
 gem 'gon'
+gem 'paperclip'
```

Comment:
@k1r8r0wn why do you add a gem in the pull request about readme adding? :confused: